lib/cc/engine/csslint/check_details.rb

```diff
@@ -0,0 +1,55 @@
+module CC
+  module Engine
+    class CSSlint
+      # https://github.com/CSSLint/csslint/wiki/Rules
+      class CheckDetails
+        DEFAULT_CATEGORY = "Style".freeze
+        DEFAULT_REMEDIATION_POINTS = 50_000.freeze
+
+        attr_reader :categories, :remediation_points
+
+        def self.all
+          @all ||= {
+            "net.csslint.Adjoiningclasses" => new(categories: "Compatability"),
```

Comment:
should these go in a constant? Maybe of the format

```ruby
"net.csslint.Adjoiningclasses" =>  { categories: "Compatability", }
```

and then in fetch below you can do

```ruby
new(all.fetch(check_name, {}))
````